first-bot/a-basic-command-handler.md

```diff
@@ -70,7 +70,7 @@ const client = new Client({
 const config = require("./config.json");
 // We also need to make sure we're attaching the config to the CLIENT so it's accessible everywhere!
 client.config = config;
-client.commands = new Discord.Collection();
+client.commands = new Collection();
```

Comment:
Discord was never shown to be required, so instead, here we destructure Collection and use that